tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```
@@ -63,24 +63,53 @@ def hard_bounce(test, topic, broker_type):
-    
+
+def bulk_clean_shutdown(test, num_failures):
+    for num in range(0, num_failures - 1):
+        signal_node(test, test.kafka.nodes[num], signal.SIGTERM)
+
+def bulk_hard_shutdown(test, num_failures):
+    for num in range(0, num_failures - 1):
+        signal_node(test, test.kafka.nodes[num], signal.SIGKILL)
+
+def bulk_clean_bounce(test, num_failures):
+    for i in range(5):
```

Ruff (0.14.1)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)